### PR TITLE
Lionelb/feat add theme in search result

### LINKED ISSUE #100: intégrer les thèmes dans la recherche
Les thèmes (en bas) doivent aussi apparaitre dans les résultats

### PR DESCRIPTION
**Pull request by lionelB**
fix #100

- index themes in code-du-travail-data (⚠️  duplicate themes.json between code-du-travail-frontend and code-du-travail-data)
- rename `code-du-travail-frontend/src/data/themes2.js` to `code-du-travail-frontend/src/data/themes2.json` 
- add excludeSource param in api to limit search result (remove search with source = courrier-type or themes
- add theme in suggested results and search (unless on page themes, where theme results are filtered)

### CODE REVIEW

**Comment by revolunet** [comment]
top merci. faut qu'on trouve une soluce pour exposer ce theme.js à l'UI. API ?

File: packages/code-du-travail-frontend/pages/theme.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ class Theme extends React.Component {
             )) ||
             null}
           {theme &&
-            theme.type !== "root" && <SearchQuery query={theme.title} />}
+            theme.type !== "root" && <SearchQuery query={theme.title} excludeSource={["themes"]} />}
```
Comment (revolunet) [inline packages/code-du-travail-frontend/pages/theme.js]: nitpick : `excludeSource` avec un `s` comme on fournit un array ?

File: packages/code-du-travail-frontend/src/search/Search.js [hunk 2]
```diff
@@ -36,9 +36,16 @@ const FormSearchButton = () => (
...
+const fetchResults = (query, endPoint = "search", excludeSource) => {
+  let urlParams = new URLSearchParams();
+  urlParams.append("q", query);
+  if (excludeSource) {
+    urlParams.append("excludeSource", excludeSource);
+  }
+  const url = `${process.env.API_URL}/${endPoint}?${urlParams.toString()}`;
+
   return fetch(url).then(response => {
+    console.log(url, response)
```
Comment (revolunet) [inline packages/code-du-travail-frontend/src/search/Search.js]: rm

**Review by revolunet — APPROVED** [review]

File: packages/code-du-travail-data/search/extraction/themes_front/data.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            text = item.get('text', '')
+            if not text:
+                logger.debug('No text found for title: %s\n%s', item['title'])
+
+            slug = ''
+            if isinstance(item['slug'], list):
+                slug = '/'.join(item['slug'])
+            else:
+                slug = item['slug']
+            logger.debug(slug)
```
Comment (revolunet) [inline packages/code-du-travail-data/search/extraction/themes_front/data.py]: oops

Comment (lionelB) [reply packages/code-du-travail-data/search/extraction/themes_front/data.py]: :'(

**Pull request merged**